src/identity/hooks/useName.ts

```diff
@@ -1,10 +1,12 @@
 import { useQuery } from '@tanstack/react-query';
-import type { Address } from 'viem';
 import { getName } from '../core/getName';
+import type { Address, Chain } from 'viem';
 import type { GetNameReturnType } from '../types';
+import type { QueryOptions } from '@tanstack/react-query';
 
 type UseNameOptions = {
   address: Address; // The Ethereum address for which the ENS name is to be fetched.
+  chain?: Chain;
```

Comment:
Very much a nit, let's add a line comment here, like we have for address, and let's move this in the `types` folder.